src/components/tooltip/ToolTip.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Normaltekst, Undertekst } from 'nav-frontend-typografi';
+import { Bold } from '../common-styled-components';
+import ShowThenHide from '../ShowThenHide';
+import { removeWhitespace } from '../../utils/string-utils';
+import * as Cookies from 'js-cookie';
+
+interface Props {
+    children: string;
+}
+
+interface State {
+    show: boolean;
+}
+
+const Style = styled.div`
+  position: fixed;
+  top: 20vh;
+  left: 50vw;
+  transform: translateX(-50%);
+  background-color: #333;
+  border: .2rem white solid;
+  border-radius: 1rem;
+  padding: 1rem 1.5rem;
+  color: white;
+  box-shadow: 0 .2rem .5rem rgba(0, 0, 0, 0.5);
+  opacity: .8;
+  pointer-events: none;
+  z-index: 1000;
+  text-align: right;
+`;
+
+const Button = styled.button`
+  background-color: transparent;
+  border: none;
+  color: white;
+  pointer-events: auto;
+  > *:hover {
+    text-decoration: underline;
+  }
+`;
+
+class ToolTip extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {show: true};
+    }
+
+    getCookieNavn() {
+        return 'tooltip' + removeWhitespace(this.props.children);
+    }
+
+    dontShowAgain() {
+        Cookies.set(this.getCookieNavn(), 'dontShow');
+    }
+
+    handleButton() {
+        this.dontShowAgain();
+        this.setState({show: false});
+    }
+
+    userHasUnsubscribed() {
+        return Cookies.get(this.getCookieNavn());
+    }
+
+    render() {
+        if (
+            !this.props.children ||
+            this.props.children === '' ||
+            !this.state.show ||
+            this.userHasUnsubscribed()
+        ) {
+            return null;
+        }
+
+        return (
+            <ShowThenHide>
```

Comment:
Forstår at det nå er to states som avgjører om denne er synlig. Ser verdien av å skille dem, da de er basert på forskjellig funksjonalitet, men kaaan det være de burde være samme state på et vis? Vet ikke selv, just spitballing.

Reply:
Og kanskje det stedet burde være redux? Er jo litt det her redux tenker å løse, flere states forskjellige steder som kombinert gir forskjellig resultater.

Reply:
kan se på det i morgen, husker ikke helt hva jeg tenkte her. Tror det har litt å gjøre med at jeg så på dette som en "animasjons-state", og det tenker jeg ikke har noen verdi å legge i redux da det ikke har noe med brukerinteraksjon å gjøre.